### PR TITLE
Add Russian translation

### PR DESCRIPTION
**Pull request by TotalCaesar659**

### CODE REVIEW

**Comment by Lakritzator** [comment]
Hey thanks!

Oh wow, I first need to figure out what we did back than, with the installer and what these files were for. We didn't have a new language in the installer for ages now.

Maybe @jklingen still knows...

Best wishes,
Robin

File: src/Greenshot/Languages/Installer/language-installer-ru-RU.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<language description="Russian" ietf="ru-RU" version="1.0.0" languagegroup="1">
+	<resources>
+		<resource name="confluence">Плагин Confluence</resource>
```
Comment (jklingen) [inline src/Greenshot/Languages/Installer/language-installer-ru-RU.xml:4]:
```suggestion
		<resource name="confluence">Плагин Confluence</resource>
		<resource name="office">Плагин Microsoft Office</resource>
```

File: src/Greenshot/Languages/Installer/language-installer-ru-RU.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+		<resource name="externalcommand">Открыть с плагином с помощью внешней команды</resource>
+		<resource name="imgur">Плагин Imgur (смотрите https://imgur.com/)</resource>
+		<resource name="jira">Плагин Jira</resource>
+		<resource name="language">Дополнительные языки</resource>
+		<resource name="ocr">Плагин OCR (требуется Microsoft Office Document Imaging (MODI))</resource>
+		<resource name="optimize">Идет оптимизация производительности, это может занять некоторое время.</resource>
+		<resource name="startgreenshot">Запустить {#ExeName}</resource>
+		<resource name="startup">Запускать {#ExeName} при старте Windows</resource>
```
Comment (jklingen) [inline src/Greenshot/Languages/Installer/language-installer-ru-RU.xml:13]:
```suggestion
		<resource name="startup">Запускать {#ExeName} при старте Windows</resource>
		<resource name="win10">Плагин Windows 10</resource>
```

**Comment by jklingen** [comment]
Note: I have already added the translations above to the installer (`develop` and `release/1.3`) - if my two added copy-paste translations are correct, we can simply merge this and no further action is needed.

**Pull request merged**